tutorials/algorithms/02_vqe_advanced_options.ipynb

```diff
@@ -0,0 +1,572 @@
+      "text/plain": [
+       "<Figure size 864x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pylab.rcParams[\"figure.figsize\"] = (12, 8)\n",
+    "pylab.plot(counts, values, label=type(optimizer).__name__)\n",
+    "pylab.xlabel(\"Eval count\")\n",
+    "pylab.ylabel(\"Energy\")\n",
+    "pylab.title(\"Energy convergence using Gradient\")\n",
+    "pylab.legend(loc=\"upper right\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Monitoring using logging\n",
+    "\n",
+    "Much of the code is instrumented with [Python logging](https://docs.python.org/3/library/logging.html) statements. The logging is configurable to adjust level of logging etc. Here we set the logging level to `INFO`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "\n",
+    "# uncomment to set up logging\n",
+    "# logging.basicConfig(level=logging.INFO)\n",
+    "# logging.getLogger(\"qiskit.algorithms.minimum_eigen_solvers.vqe\").setLevel(logging.INFO)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "and at INFO level logging VQE will include information on the evaluations, as below.\n",
+    "\n",
+    "INFO:qiskit.algorithms.minimum_eigen_solvers.vqe:Energy evaluation returned [-1.07392554] - 116.41884 (ms), eval count: 1\n",
+    "INFO:qiskit.algorithms.minimum_eigen_solvers.vqe:Energy evaluation returned [-1.43698938] - 4.05884 (ms), eval count: 2\n",
+    "INFO:qiskit.algorithms.minimum_eigen_solvers.vqe:Energy evaluation returned [-1.74596698] - 7.40194 (ms), eval count: 3\n",
+    "INFO:qiskit.algorithms.minimum_eigen_solvers.vqe:Energy evaluation returned [-1.75399268] - 6.61016 (ms), eval count: 4"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Initial point\n",
+    "\n",
+    "The `initial_point`option allows the optimization to begin at a given point, defined by a list of parameter values that will configure the ansatz. By default, the initial point is `None`, which means that VQE will choose it based on the initial state of the ansatz. If the ansatz does not provide a preferred state, a random initial point that fits within the ansatz bounds will be chosen. If an initial point is supplied by the user, it will take priority and be used - note though it must match in length to the number of parameters in the ansatz circuit.\n",
```

Comment:
In terms of the initial point I think nowadays its either a randomly chosen point or explicitly chosen/input by a user.

We used to have the possibility that an anstaz could indicate a preferred initial point, if it were given an initial state. For instance HF + UCC would have done this in the past. This is no longer the case. Yes it still will look to the ansatz in case it has any parameter bounds defined, but that is just to ensure that the random point will be within these bounds and hence valid. I think this is more than a user needs to know though and just saying its random by default should suffice.

Reply:
This makes sense, I was a bit confused about what "preferred by the ansatz" meant. I see why now.